common/constants.go

```diff
@@ -48,7 +48,7 @@ const (
 	NeptuneClusterRoleName               = "neptune-manager-role"
 	NeptuneMetricServicePort             = 8443
 	NeptuneMetricServiceProtocol         = "TCP"
-	NeptuneDefaultImage                  = "controller:9028cae"
+	NeptuneDefaultImage                  = "controller:4d70cfa"
```

Comment:
I assume this is ok, right? when we release, this becomes something like v1.0.0? I mean this is meant to be hard coded per release?